resolwe/flow/models/annotations.py

```diff
@@ -370,6 +370,15 @@ def __str__(self):
 class AnnotationValue(AuditModel):
     """The value of the annotation."""
 
+    class Meta:
```

Comment:
This can/should go into a separate commit.

Reply:
Done.